[2024-04-17T00:22:40.342+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-17T00:22:40.430+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: m_dag_ejecutaapt.ejecutar_script_ scheduled__2024-04-14T00:00:00+00:00 [queued]>
[2024-04-17T00:22:40.509+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: m_dag_ejecutaapt.ejecutar_script_ scheduled__2024-04-14T00:00:00+00:00 [queued]>
[2024-04-17T00:22:40.510+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 2
[2024-04-17T00:22:40.659+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): ejecutar_script_> on 2024-04-14 00:00:00+00:00
[2024-04-17T00:22:40.761+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=303) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-04-17T00:22:40.769+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'm_dag_ejecutaapt', 'ejecutar_script_', 'scheduled__2024-04-14T00:00:00+00:00', '--job-id', '284', '--raw', '--subdir', 'DAGS_FOLDER/holaa.py', '--cfg-path', '/tmp/tmpyxe1p4q0']
[2024-04-17T00:22:40.794+0000] {standard_task_runner.py:91} INFO - Job 284: Subtask ejecutar_script_
[2024-04-17T00:22:40.801+0000] {standard_task_runner.py:63} INFO - Started process 306 to run task
[2024-04-17T00:22:46.602+0000] {task_command.py:426} INFO - Running <TaskInstance: m_dag_ejecutaapt.ejecutar_script_ scheduled__2024-04-14T00:00:00+00:00 [running]> on host 016d8c81e0c3
[2024-04-17T00:22:47.982+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='usuario' AIRFLOW_CTX_DAG_ID='m_dag_ejecutaapt' AIRFLOW_CTX_TASK_ID='ejecutar_script_' AIRFLOW_CTX_EXECUTION_DATE='2024-04-14T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-14T00:00:00+00:00'
[2024-04-17T00:22:48.008+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-17T00:22:48.060+0000] {logging_mixin.py:188} INFO - Hola Mundo
[2024-04-17T00:22:48.061+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-04-17T00:22:48.063+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-17T00:22:48.096+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=m_dag_ejecutaapt, task_id=ejecutar_script_, execution_date=20240414T000000, start_date=20240417T002240, end_date=20240417T002248
[2024-04-17T00:22:48.363+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-17T00:22:48.496+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-17T00:22:48.536+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
